instrumentation/pekko/pekko-http-1.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/pekkohttp/v1_0/server/PekkoHttpServerTracer.java

```diff
@@ -143,6 +145,14 @@ public void onPush() {
                 if (!headers.isEmpty()) {
                   response = (HttpResponse) response.addHeaders(headers);
                 }
+                PekkoRouteHolder routeHolder = tracingRequest.context.get(PekkoRouteHolder.KEY);
+                if (routeHolder != null) {
+                  routeHolder.pushIfNotCompletelyMatched("**");
```

Comment:
why not just `*`

Reply:
There isn't really a standard way these routes should be constructed. It could be the path template used by the framework, such as jax-rs `@Path`, or some custom interpretation like here. In my opinion it is easier to just interpret `*` as any number of characters, anything more complicated and we'd probably need to document it. User can always look at the full path of the request if needed. To me `**` looks like a typo, not that in spring-mvc route can also end with `**`.